planetiler-core/src/main/java/com/onthegomap/planetiler/ForwardingProfile.java

```diff
@@ -81,13 +81,21 @@ protected ForwardingProfile(Handler... handlers) {
   }
 
   private boolean caresAboutLayer(String layer) {
-    return (onlyLayers.isEmpty() || onlyLayers.contains(layer)) && !excludeLayers.contains(layer);
+    return ((onlyLayers.isEmpty() || onlyLayers.contains(layer)) && !excludeLayers.contains(layer)) ||
+      this.handlers.stream()
+        .filter(h -> h instanceof HandlerForLayer)
+        .map(h -> (HandlerForLayer) h)
```

Comment:
sonar wants this to be `.filter(HandlerForLayer.class::isInstance).map(HandlerForLayer.class::cast)` but I think the way you have it is more readable so feel free to ignore sonar here.

Reply:
Adjusted.